docs/tools/cli/service/privatelink.rst

```diff
@@ -0,0 +1,423 @@
+    ============================================================================================  =========================================================================================================================================================================================  ========  ====================================
+    aivenprod-ss123456789ab.12345678-90ab-cdef-9876-543210abcdef.eastus.azure.privatelinkservice  /subscriptions/12345678-90ab-cdef-1234-567890abcdef/resourceGroups/aivenprod-12345678-90ab-cdef-1234-567890abcdef/providers/Microsoft.Network/privateLinkServices/aivenprod-ss123456789ab  deleting  12345678-90ab-cdef-0987-6543210abcde
+
+.. _avn_service_privatelink_azure_get:
+
+``avn service privatelink azure get``
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Lists PrivateLink information for a service.
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The name of the service
+  * - ``--project``
+    - The project to fetch details for
+  * - ``--format``
+    - Format of the output string
+
+**Example:** List PrivateLink information for ``kafka-12a3b4c5`` service.
+
+::
+
+    avn service privatelink azure get kafka-12a3b4c5
+
+An example of output:
+
+.. code:: text
+
+    AZURE_SERVICE_ALIAS                                                                           AZURE_SERVICE_ID                                                                                                                                                                           STATE   USER_SUBSCRIPTION_IDS               
+    ============================================================================================  =========================================================================================================================================================================================  ======  ====================================
+    aivenprod-ss123456789ab.12345678-90ab-cdef-9876-543210abcdef.eastus.azure.privatelinkservice  /subscriptions/12345678-90ab-cdef-1234-567890abcdef/resourceGroups/aivenprod-12345678-90ab-cdef-1234-567890abcdef/providers/Microsoft.Network/privateLinkServices/aivenprod-ss123456789ab  active  12345678-90ab-cdef-0987-6543210abcde
+
+.. _avn_service_privatelink_azure_refresh:
+
+``avn service privatelink azure refresh``
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Refreshes incoming endpoint connections.
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The name of the service
+  * - ``--project``
+    - The project to fetch details for
+  * - ``--format``
+    - Format of the output string
```

Comment:
Can we have a code sample?

Reply:
:+1: I'll add it, however, this command doesn't produce output.